src/components/whiteboard/SofaWhiteboard.tsx

```
@@ -0,0 +1,150 @@
+          ref={canvasRef}
+          className="w-full h-full block touch-none"
+          width={WHITEBOARD_CONSTANTS.INTERNAL_WIDTH}
+          height={WHITEBOARD_CONSTANTS.INTERNAL_HEIGHT}
+          onMouseDown={startDrawing}
+          onMouseUp={() => stopDrawing()}
+          onMouseOut={() => stopDrawing()}
+          onMouseMove={(e) => draw(e)}
+          onTouchStart={startDrawing}
+          onTouchEnd={() => stopDrawing()}
+          onTouchMove={(e) => draw(e)}
+        />
```

Biome (2.4.4)

[error] onMouseOut must be accompanied by onBlur for accessibility.

(lint/a11y/useKeyWithMouseEvents)